internal/rbd/controllerserver.go

```diff
@@ -968,8 +993,9 @@ func (cs *ControllerServer) CreateSnapshot(
 
 		return nil, status.Errorf(codes.Internal, err.Error())
 	}
+	parameters := req.GetParameters()
```

Comment:
snapshotcontent can be detatched and reattached to a new snapshot in a different namespace. do we have code to regenerate the metadata in that case?

Reply:
I have not checked this part, AI on me

Reply:
lets open up a new issue to track it. we need new controller for this one. we can do a follow up